src/v2/Utils/getSaleOrLotTimerInfo.ts

```diff
@@ -0,0 +1,91 @@
+export interface Time {
+  days: string
+  hours: string
+  minutes: string
+  seconds: string
+}
+
+interface TimerInfo {
+  copy: string
+  color: string
+}
+
+// If `extendedBiddingEndAt` is passed in and non-null, the `time`
+// parameter refers to the time left in the extended period.
+export const getSaleOrLotTimerInfo = (
+  time: Time,
+  hasStarted?: boolean,
+  saleHasEnded?: boolean,
+  isSaleInfo: boolean = false,
+  extendedBiddingEndAt?: string | null
+): TimerInfo => {
+  const { days, hours, minutes, seconds } = time
+
+  const parsedDays = parseInt(days, 10)
+  const parsedHours = parseInt(hours, 10)
+  const parsedMinutes = parseInt(minutes, 10)
+  const parsedSeconds = parseInt(seconds, 10)
+
+  let copy = ""
+  let color = "blue100"
+
+  // Sale has not yet started
+  if (!hasStarted) {
+    if (parsedDays < 1) {
+      copy = "Bidding Starts Today"
+      // Entered extended bidding
+    } else if (extendedBiddingEndAt) {
+      copy = `Extended: ${parsedMinutes}m ${parsedSeconds}s`
+      color = "red100"
+    } else {
+      copy = `${parsedDays} Day${
+        parsedDays > 1 ? "s" : ""
+      } Until Bidding Starts`
+    }
+    // Sale has started
+  } else {
+    // When the timer is on the sale:
+    if (isSaleInfo) {
+      if (saleHasEnded) {
+        copy = "Lots are closing"
+        color = "red100"
+        // More than 24 hours until close
+      } else if (parsedDays >= 1) {
+        copy = `${parsedDays + 1} Day${
+          parsedDays >= 1 ? "s" : ""
+        } Until Lots Start Closing`
+      }
+      // 1-24 hours until close
+      else if (parsedDays < 1 && parsedHours >= 1) {
+        copy = `${parsedHours}h ${parsedMinutes}m Until Lots Start Closing`
+        color = "red100"
+      }
+
+      // <60 mins until close
+      else if (parsedDays < 1 && parsedHours < 1) {
+        copy = `${parsedMinutes}m ${parsedSeconds}s Until Lots Start Closing`
+        color = "red100"
+      }
+
+      // When the timer is on the lot:
+    } else {
+      // More than 24 hours until close
+      if (parsedDays >= 1) {
+        copy = `${parsedDays}d ${parsedHours}h`
+      }
+
+      // 1-24 hours until close
+      else if (parsedDays < 1 && parsedHours >= 1) {
+        copy = `${parsedHours}h ${parsedMinutes}m`
+      }
+
+      // <60 mins until close
+      else if (parsedDays < 1 && parsedHours < 1) {
+        copy = `${parsedMinutes}m ${parsedSeconds}s`
+        color = "red100"
```

Comment:
I think this red state on the lot should just be when it's < 2 minutes (going from `LotTimer`).

Reply:
No I had it wrong before. Timer should be red under 1 hour: https://www.figma.com/file/PD7gXzuexzbi4Zz1tZLssx/Popcorn-and-cascading-bidding?node-id=902%3A7045

Reply:
From Figma:

<img width="413" alt="Screen Shot 2022-04-27 at 7 56 23 PM" src="https://user-images.githubusercontent.com/9466631/165766601-19057b57-545c-45ca-ba08-d0059a78c506.png">